db/migrate/14_drop_openstax_uid_and_username_uniqueness.rb

```diff
@@ -0,0 +1,9 @@
+class DropOpenStaxUidAndUsernameUniqueness < ActiveRecord::Migration[5.2]
+  def change
+    remove_index :openstax_accounts_accounts, column: [ :openstax_uid ], unique: true
+    remove_index :openstax_accounts_accounts, column: [ :username ], unique: true
+
+    add_index :openstax_accounts_accounts, :openstax_uid
+    add_index :openstax_accounts_accounts, :username
+  end
+end
```

Comment:
should we add indexes in background?

Reply:
adding indexes is safe, and only slow if it's a millions of rows table. I think we're ok with these.

Reply:
I think it'll be fine, we don't have that many accounts